packages/visual-editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@contentful/experience-builder-visual-editor",
-  "version": "0.0.2-alpha.26",
+  "name": "@contentful/experiences-visual-editor-react",
+  "version": "0.0.1-alpha.25",
```

Comment:
Guess we didn't check all the packages to find the latest version. The rest are on `alpha.25` do I need to have an extra PR at the end so we can backtrack and update all the package.json's to have `alpha.26`?

Reply:
The alpha version is defined per package and went out of sync when only certain but not all packages were updated. This makes sense as when you only change something in the visual-editor you don't want to create a new release for `core`.
To avoid starting the new package with `alpha.25`, I suggest starting here again with `0.0.1-alpha.1`. As we do this for all packages, they would be at least for now in sync again :)